### PR TITLE
fix 176591 adslgate.com

### LINKED ISSUE #176591: www.adslgate.com
### Issue URL (Ads)
 [https://www.adslgate.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.adslgate.com)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.35
System version: | Windows 10
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances, <br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>EasyList Cookie List<br/><br/><b>Other:</b><br/>Filter unblocking search ads and self-promotion<br/><br/><details><summary><b>Language-specific:</b></summary>AdGuard Russian, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Spanish/Portuguese, <br/>AdGuard Turkish, <br/>AdGuard French, <br/>Official Polish filters for AdBlock, uBlock Origin & AdGuard, <br/>AdGuard Chinese, <br/>ABPindo, <br/>Bulgarian list, <br/>EasyList Italy, <br/>Liste AR, <br/>EasyList Thailand, <br/>ABPVN List, <br/>List-KR, <br/>Persian Blocker, <br/>Polish Anti Adblock Filters, <br/>Frellwit's Swedish Filter, <br/>Dandelion Sprout's Nordic Filters</details>

### PR DESCRIPTION
**Pull request by maximtop**
fix #176591

### CODE REVIEW

File: BaseFilter/sections/foreign.txt [hunk 1]
```diff
@@ -608,6 +608,7 @@ gerasanews.com##.iads_border
 qatarshares.com##.above_body > div#header+table[width="100%"][cellpadding="0"]
 adslgate.com##.smallfont > center > a > img
 adslgate.com##.smallfont a[href^="http://deals.souq.com/"]
+adslgate.com##.skyscraper-banner-right
```
Comment (zloyden) [inline BaseFilter/sections/foreign.txt:611]: Could you please move the rule next to the comment `! NOTE: Arabian` with the task link?
Like
```adblock
! https://github.com/AdguardTeam/AdguardFilters/issues/176591
adslgate.com##.skyscraper-banner-right
```

Comment (zloyden) [reply BaseFilter/sections/foreign.txt:611]: Please also check if the current rules are obsolete or not.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: BaseFilter/sections/foreign.txt (hunk 1 above)
Comment (Alex-302) [reply BaseFilter/sections/foreign.txt:611]: I think it is not a problem - the rule is not dangerous.

Comment (zloyden) [reply BaseFilter/sections/foreign.txt:611]: Ok.

Comment (Alex-302) [reply BaseFilter/sections/foreign.txt:611]: Reviewing of old rules is also not a task of this event)

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**